train.py

```diff
@@ -0,0 +1,253 @@
+import pathlib
+import json
+import string
+import re
+import pickle
+import argparse
+import numpy as np
+from gensim.models import Word2Vec
+from keras.api import Sequential
+from keras.api.layers import Dense, LSTM, Dropout, Embedding, Bidirectional, Conv1D, MaxPooling1D
+from keras.api.optimizers import Adam
+from keras.api.callbacks import EarlyStopping, ReduceLROnPlateau
+from keras.api.preprocessing.sequence import pad_sequences
+from tensorflow.keras.preprocessing.text import Tokenizer
+from sklearn.preprocessing import LabelEncoder
+import nltk
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+
+nltk.download('punkt')
```

Comment:
putting this code here means it will get executed when `train` or anything from the module is imported into another module. And you probably don't want to trigger a download just by importing something